ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-release-ocm-2.6.yaml

```diff
@@ -117,6 +117,8 @@ tests:
       INDEX_IMAGE: assisted-service-index
       OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
       PROVIDER_IMAGE: pipeline:cluster-api-provider-agent
+    env:
+      ALLOW_CONVERGED_FLOW: "False"
```

Comment:
should be applied to the periodic jobs as well? ^

Reply:
Added